eve/workers/pod.yaml

```diff
@@ -72,9 +84,7 @@ spec:
         - bash
         - -ec
         - |
-          # wait for exports to be created
-          sleep 15 
-          source /root/.aws/exports || exit 1
+          sleep 10 # wait for mongo
```

Comment:
note the workaround to fix CI flakyness when s3 container start before mongodb

Reply:
Yeah I created a ticket for the mongodb flakiness last week. It's been a big/annoying problem.
https://scality.atlassian.net/browse/ZENKO-607

Reply:
Damn, I have this magic power of always creating duplicates... https://scality.atlassian.net/browse/S3C-1567

I'll close mine, thanks!